prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -379,12 +417,27 @@ class FHIRRouter(
             defaultResponse
         } else {
             filter.all {
-                FhirPathUtils.evaluateCondition(
+                val longhand = replaceShorthand(it)
```

Comment:
This val's name is confusing.  Use a more descriptive name

Reply:
i like the name!
fixed tho, and another tweak made the val not needed at all